sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -279,11 +279,11 @@ class CodegenContext {
       inlineToOuterClass: Boolean = false): String = {
     // The number of named constants that can exist in the class is limited by the Constant Pool
     // limit, 65,536. We cannot know how many constants will be inserted for a class, so we use a
-    // threshold of 1600k bytes to determine when a function should be inlined to a private, nested
-    // sub-class.
+    // threshold to determine when a function should be inlined to a private, nested sub-class
+    val generatedClassLengthThreshold = SQLConf.get.generatedClassLengthThreshold
```

Comment:
Good to know [this discussion](https://github.com/apache/spark/pull/19449#pullrequestreview-67789784)

Reply:
thanks for the interesting link. I assume using SparkEnv has the same problem, hasn't it?